RESTier/TripPinInMemory/Microsoft.OData.Service.Sample.TrippinInMemory/Models/Person.cs

```diff
@@ -53,5 +53,14 @@ public class Person
         public Feature FavoriteFeature { get; set; }
 
         public virtual ICollection<Feature> Features { get; set; }
+
+        public Person()
+        {
+            Emails = new List<string>();
```

Comment:
to create/allocate empty list for those collection properties looks 'annoying'.

I know OData spec requires collection should not be null. But, Why? can we change this 'Rule'?

From C# perspective, a collection value call be:
1) null
2) empty
3) items

Reply:
IMO it has nothing to do with OData. Best practices on dealing with objects with collections is to initialize them by default. When a developer goes to work with this object model and creates a new Person, they shouldn't have to KNOW that they _also_ have to create a new List for the Emails... it should just be there ready for you to add stuff to it.